tests/test-ldouble/test_ldouble.ml

```diff
@@ -116,6 +116,18 @@ let test_conv _ =
   List.iter (fun a -> assert_bool "to/of_float" (a = LDouble.(to_float (of_float a)))) flts;
   assert_bool "to_int" (3 = LDouble.(to_int (of_float 3.45)));
   assert_bool "to_int" (-34 = LDouble.(to_int (of_float (-34.999))));
+  if Sys.word_size = 32 then (
+    let mi = float_of_int min_int in
+    let ma = float_of_int max_int in
+    assert_bool "to_int" (min_int = LDouble.(to_int (of_float mi)));
+    assert_bool "to_int" (max_int = LDouble.(to_int (of_float ma)));
+  )
+  else (
+    let mi = -9007199254740991. in
+    let ma =  9007199254740991. in
```

Comment:
I wonder if it's worth the hassle of exposing `LDBL_MANT_DIG` or similar in the `LDouble` module, to better test platforms that have a `long double` distinct from `double`.  I think it would be good to at least have a comment to the effect that this is 2⁵³-1, and perhaps also note the guarantee that we can represent integers within that range precisely.

Reply:
Ok, I've added `LDouble.mant_dig` that is now also used inside the test.